applications/openshift/ocp-permissions/ocp-files/file_permissions_master_openvswitch/rule.yml

```diff
@@ -30,5 +30,5 @@ template:
     name: file_permissions
     vars:
         filepath: /etc/origin/openvswitch/
-        file_regex: ^[^\.].*$
+        file_regex: ^[^.].*$
```

Comment:
This doesn't work as intended for the Bash remediation.

Unlike in Ansible and OVAL, in Bash the `file_regex` is matched against the whole path, not just the base name.
`-regex` from `find` is matched against whole path. `-name` could be used, but it takes a shell pattern, not a regex.
https://github.com/ComplianceAsCode/content/blob/master/shared/templates/template_BASH_file_permissions#L7